workspaces/ui-v2/src/optic-components/pages/docs/components/EditContributionsButton.tsx

```diff
@@ -50,17 +50,18 @@ export function EditContributionsButton() {
       >
         {contents}
       </ToggleButton>
-      <CommitMessageModal
```

Comment:
Changing this so it gets remounted such that state is reset each time we open + close